src/Solvers/fourier_tridiagonal_poisson_solver.jl

```diff
@@ -43,7 +43,7 @@ end
     Nz = size(grid, 3)
 
     # Using a homogeneous Neumann (zero Gradient) boundary condition:
-    @inbounds D[i, j, 1] = -1 / Δzᵃᵃᶠ(i, j, 2, grid) - Δzᵃᵃᶜ(i, j, 1, grid) * (λx[i] + λy[j])
+    @inbounds D[i, j, 1] = -1 / Δzᵃᵃᶠ(i, j, 2, grid) - Δzᵃᵃᶜ(i, j, 1, grid) * (λx[i] + λy[j]) + (1 / (g * Δt^2))
```

Comment:
Nice. You'll have to hard code your `g` and `Δt` here since they aren't available. When we adapt this code to be merged into main we will design an interface that allows this to be input upon model construction. We will also have to forbid adaptive time-stepping, or better yet implement a way to update the pressure solver if the time-step changes.